app/controllers/schools_controller.rb

```diff
@@ -165,66 +165,48 @@ def authorized_students_for_overview(school)
     end
   end
 
-  # Methods for Dashboard
-
-  def individual_student_dashboard_data(student)
-    # This is a temporary workaround for New Bedford
-    homeroom_label = student.try(:homeroom).try(:educator).try(:full_name) || student.try(:homeroom).try(:name)
-    HashWithIndifferentAccess.new({
-      first_name: student.first_name,
-      last_name: student.last_name,
-      id: student.id,
-      grade: student.grade,
-      race: student.race,
-      homeroom_label: homeroom_label,
-      absences: student.dashboard_absences,
-      tardies: student.dashboard_tardies,
-      event_notes: student.event_notes
-    })
-  end
+  # Methods for Dashboards #
 
   def individual_student_absence_data(student)
-    # Gathers only the information needed for a specific dashboard view.
-    homeroom_label = student.try(:homeroom).try(:educator).try(:full_name) || student.try(:homeroom).try(:name)
-    HashWithIndifferentAccess.new({
+    {
       first_name: student.first_name,
       last_name: student.last_name,
       id: student.id,
-      homeroom_label: homeroom_label,
+      homeroom_label: homeroom_label(student.homeroom),
       absences: student.dashboard_absences,
-      event_notes: student.event_notes
-    })
+      sst_notes: student.sst_notes,
+    }
   end
 
   def individual_student_tardies_data(student)
-    # Gathers only the information needed for a specific dashboard view.
-    homeroom_label = student.try(:homeroom).try(:educator).try(:full_name) || student.try(:homeroom).try(:name)
-    HashWithIndifferentAccess.new({
+    {
       first_name: student.first_name,
       last_name: student.last_name,
       id: student.id,
-      homeroom_label: homeroom_label,
+      homeroom_label: homeroom_label(student.homeroom),
       tardies: student.dashboard_tardies,
-      event_notes: student.event_notes
-    })
+      sst_notes: student.sst_notes,
+    }
   end
 
   def individual_student_discipline_data(student)
-    # Gathers only the information needed for a specific dashboard view.
-    homeroom_label = student.try(:homeroom).try(:educator).try(:full_name) || student.try(:homeroom).try(:name)
-    HashWithIndifferentAccess.new({
+    {
       first_name: student.first_name,
       last_name: student.last_name,
       id: student.id,
-      homeroom_label: homeroom_label,
+      homeroom_label: homeroom_label(student.homeroom),
       grade: student.grade,
       race: student.race,
       discipline_incidents: student.discipline_incidents,
-      event_notes: student.event_notes
-    })
+      sst_notes: student.sst_notes,
+    }
   end
 
-  def students_for_dashboard(school)
+  def active_students(school)
     school.students.active
   end
+
+  def homeroom_label(homeroom)
+    homeroom.try(:educator).try(:full_name) || homeroom.try(:name) || "No Homeroom"
```

Comment:
This is probably scope creep for this PR, but . . . I figured as long as we're testing for whether or not the student has a homeroom label here, why not return the string value for when there's no homeroom? That lets us remove JS code in a couple of places in the dashboards testing for no-homeroom.